### PR TITLE
feat: add support for pnpm

### PR DESCRIPTION
**Pull request by mcansh**
Signed-off-by: Logan McAnsh <logan@mcan.sh>

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const childProcess = require("child_process");
 const version = process.argv[2] || "latest";
 
 const packageJsonPath = path.join(process.cwd(), "package.json");
-if (!fs.existsSync(path.join(process.cwd(), "package-lock.json"))) {
+if (!fs.existsSync(packageJsonPath)) {
```
Comment (brophdawg11) [inline index.js:10]: doh!  Good catch - that's what I get for literally never using `yarn` in my own apps

**Review by brophdawg11 — APPROVED** [review]
Thanks Logan!

File: index.js [hunk 2]
```diff
@@ -27,7 +27,11 @@ const implementations = {
   },
   yarn: {
     install: (packages, isDev) => `yarn add ${isDev ? "-D" : ""} ${packages}`,
-    sync: "yarn",
+    sync: "yarn install --frozen-lockfile",
```
Comment (rossipedia) [inline index.js:30]: `--frozen-lockfile` is deprecated in current versions of yarn (at least as of v3.5.0). It'll still work, it just gives a warning.

Comment (brophdawg11) [reply index.js:30]: haha, I'm on version 1.22 still 😬 .  I guess we could check `yarn --version` and use it accordingly

**Pull request merged**